hw02/Homwork 2.Rmd

```diff
@@ -0,0 +1,137 @@
+---
+title: "Homework 2"
+author: "Samruddhi Shinde"
+date: "2/7/2021"
+output:
+  html_document: default
+  github_document: default
+---
+#Exercise 1
+
+##1.1 
+
+Loading in packages needed for assignments
+```{r}
+library(tidyverse)
+library(gapminder)
+```
+
+Filter data set to show 3 countries (Mexico, Pakistan, & Zimbabwe) from the 1970s
+```{r}
+gapminder %>%
+  filter(country %in% c("Mexico","Pakistan", "Zimbabwe") & year %in% c(1970:1979))
+```
+
+##1.2
+
+Select country and gdpPercap from filtered dataset
+```{r}
+gapminder %>%
+  filter(country %in% c("Mexico","Pakistan", "Zimbabwe") & year %in% c(1970:1979)) %>%
+  select(country, gdpPercap)
+```
+
+##1.3 
+
+Create new variable for thr change in life expectancy
+```{r}
+gapminder %>%
+  mutate(change_in_lifeExp = lifeExp - lag(lifeExp)) %>%
+  filter(change_in_lifeExp<0)
+```
+
+##1.4
+
+Filter gapminder dataset to show the max GDP per capita for each country
+```{r}
+gapminder %>%
+  group_by(country) %>%
+  summarize(max(gdpPercap))
```

Comment:
I guess this works ... I'm not sure you needed to summarize the data? You could use the filter function instead.